Drivers/common/circular_buffer/src/circular_buffer.cpp

```diff
@@ -69,4 +69,8 @@ bool CircularBuffer::write(uint8_t byte) {
 
 uint16_t CircularBuffer::getFreeSpaceBytes() { 
     //NOT IMPLEMENTED YET
-    return -1; }
\ No newline at end of file
+    return -1; }
+
+uint16_t CircularBuffer::getSize_(){
```

Comment:
could you rename this method to be `getSize` to get the consistency with other methods?